src/config/topics.js

```diff
@@ -380,31 +380,30 @@ export const topics = {
   boligpriser: {
```

Comment:
what about the rest of the file? You changed the one we are using now, but it will break every single other topics-page when we start using them.